src/tests/unit/routeSpec/RouteSpec.test.js

```diff
@@ -0,0 +1,132 @@
+'use strict'
+
```

Comment:
If this PR is accepted, the next step will be to test the functionality via the handler test